nixtlats/data/tsdataset.py

```diff
@@ -176,53 +164,33 @@ def __init__(self,
         # Balances panel and creates
         # numpy  s_matrix of shape (n_series, n_s)
         # numpy ts_tensor of shape (n_series, n_channels, max_len) n_channels = t_cols + masks
-        self._create_tensor()
-
-        # Defining windows attributes by model
-        self.windows_size: int
-        self.padding: Tuple[int, int]
-
-        self._define_attributes_by_mode()
+        self.len_series, self.ts_tensor = self._create_tensor()
 
         # Defining sampleable time series
         self.ts_idxs = np.arange(self.n_series)
-        self.samplable_ts_idxs: np.ndarray
-        self.n_samplable_ts: int
+        self.sampleable_ts_idxs: np.ndarray
+        self.n_sampleable_ts: int
 
-        self._define_samplable_ts_idxs()
+        self._define_sampleable_ts_idxs()
 
 # Cell
 @patch
-def _define_attributes_by_mode(self: TimeSeriesDataset):
-    if self.mode in ['simple']:
-        self.windows_size = self.input_size + self.output_size
-        self.padding = (self.input_size, self.output_size)
-    elif self.mode in ['full']:
-        self.windows_size = self.len_sample_chunks
-        self.padding = (0, 0)
-    else:
-        raise Exception(f'There is no window strategy for {self.mode}')
-
-    assert self.windows_size <= self.window_sampling_limit, (
-        'Window sampling limit should be at least input_size + output_size '
-        'for simple mode or at least len_sample_chunks for full mode.'
-    )
+# TODO: think this. porque sample_mask? no cuenta el input... ademas que pasa con random validation...
+def _define_sampleable_ts_idxs(self: BaseDataset):
+    # sum_sample_mask = self.ts_tensor[:, self.t_cols.index('sample_mask')] \
+    #                       .sum(axis=1)
+    # if self.complete_inputs:
+    #     min_mask = self.windows_size
+    # else:
+    #     min_mask = self.output_size
+    # self.samplable_ts_idxs = np.argwhere(sum_sample_mask > min_mask).reshape(1, -1)[0]
+    # self.n_samplable_ts = self.samplable_ts_idxs.size
```

Comment:
It might be better to remove these lines rather than just commenting them.

Reply:
This filtering function seems unused, I don't know if we want to preserve it.